environment.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Bare minimum setup of the environment to use Vanilla's classes.
+ *
+ * @author Alexandre (DaazKu) Chouinard <alexandre.c@vanillaforums.com>
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+if (PHP_VERSION_ID < 50600) {
+    die('Vanilla requires PHP 5.6 or greater.');
+}
+
+// Define the constants we need to get going.
+if (!defined('APPLICATION')) {
+    define('APPLICATION', 'Vanilla');
+}
+if (!defined('APPLICATION_VERSION')) {
+    define('APPLICATION_VERSION', '2.4.200');
+}
+if (!defined('DS')) {
+    define('DS', DIRECTORY_SEPARATOR);
+}
+if (!defined('PATH_ROOT')) {
+    define('PATH_ROOT', __DIR__);
```

Comment:
Vanilla currently establishes `PATH_ROOT` by using `getcwd`. Switching this up with `__DIR__` is going to change the way it works with symlinks.

For example, say I have a dir with a Git repo: /srv/src. I symlink certain directories and files from there into a separate directory to act as a site root: /srv/html. If I'm setting `PATH_ROOT` with `__DIR__`, it's going to get set to /srv/src. If I'm setting `PATH_ROOT` with `getcwd`, it gets set to /srv/html.

Switching up to `__DIR__` might be an improvement, but there is a subtle difference.